enginetest/queries/tpcc_plans.go

```diff
@@ -49,6 +49,62 @@ SELECT c_discount, c_last, c_credit, w_tax FROM customer2, warehouse2 WHERE w_id
 			"                 ├─ name: customer2\n" +
 			"                 └─ columns: [c_id c_d_id c_w_id c_last c_credit c_discount]\n" +
 			"",
+		ExpectedEstimates: "Project\n" +
```

Comment:
I think the majority of our plan tests could be replaced with the simpler format imo. I get annoyed editing `join_op_tests.go` as often as I do, but that suite has been enough for me to not make mistakes refactoring join costing so far. So all of the TPC-X plans, and any join plans in the regular suite as a simpler format would be better. I like the duplication for integration plans, but I'd maybe put them in a different file to make them easier to review in GitHub? The differences just get so long that they are hard to review.

Reply:
Good point. I previously had it so that these tests used the full debug output, but that's not really necessary for the EXPLAIN tests, so I fixed it.

I'll look into paring down info that's in the non-debug plans but might not be needed her, like Projects and columns and ranges and whatnot.